### PR TITLE
Add Graal 19.0.0 and remove Graal RC14

### PR DESCRIPTION
**Pull request by ilopmar**
This PR add Graal 19.0.0 and removes old RC14. We'll keep RC15 and 16 for a while because the new one introduced some breaking changes.

### CODE REVIEW

File: src/main/scala/io/sdkman/changelogs/JavaMigrations.scala [hunk 1]
```diff
@@ -957,4 +957,27 @@ class JavaMigrations {
     Seq(Linux64, MacOSX, Windows).foreach(removeVersion("java", "13.ea.19-open", _))
   }
 
+  @ChangeSet(order = "138", id = "138-add_graalvm_19_0_0", author = "ilopmar")
+  def migrate138(implicit db: MongoDatabase) = {
+    List(
+      Version(
+        candidate = "java",
+        version = "19.0.0-graal-ce",
```
Comment (marc0der) [inline src/main/scala/io/sdkman/changelogs/JavaMigrations.scala]: We'll need to stick to using the same `-grl` suffix as before in order for the _sdkman-hooks_ service to handle these versions.

Comment (ilopmar) [reply src/main/scala/io/sdkman/changelogs/JavaMigrations.scala]: Pushed the change. Feel free to squash the commits when merging them

Comment (marc0der) [reply src/main/scala/io/sdkman/changelogs/JavaMigrations.scala]: Thanks @ilopmar. You have some serious Scala skills there!!! :laughing:

Comment (marc0der) [reply src/main/scala/io/sdkman/changelogs/JavaMigrations.scala]: Sorry for being a pain, it needs to be `-grl` without the `-ce` for the hooks to be picked up.

Comment (ilopmar) [reply src/main/scala/io/sdkman/changelogs/JavaMigrations.scala]: I'm not so sure, 3 commits to make a simple change :(

Comment (marc0der) [reply src/main/scala/io/sdkman/changelogs/JavaMigrations.scala]: You did well, thanks for doing this! :metal:

**Comment by marc0der** [comment]
Thank you!

**Comment by ilopmar** [comment]
Thanks buddy!

**Pull request merged**